db/migrations/4_set_username_field_in_users.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:users) do
+      rename_column :user_name, :username
```

Comment:
Why we want to rename it?

Reply:
```ruby
module Telegram
  module Bot
    module Types
      class User < Base
        attribute :id, Integer
        attribute :is_bot, Boolean
        attribute :first_name, String
        attribute :last_name, String
        attribute :username, String
        attribute :language_code, String
      end
    end
  end
end
```
This is `User` type, which is defined by `gem` and got from `message.reply_to_message.from`.
In this PR user from our database be consistent with Telegram user.